### DISCUSSION TITLE
Join with delay?

### DISCUSSION THREAD

**Discussion by swrobel** [justdan96/tsMuxer#826] [Q&A] [answered]
Is it possible to join two audio tracks with a delay between them, like one can set a delay for the start of the initial audio track?

**Comment by jcdr428** [comment, marked as answer]
It is not directly possible, however you can mux the tracks separately with delays then join them.

**Reply by swrobel** [reply]
Thank you for the reply, and my apologies for creating this as an issue instead of a discussion!
